files/IaPgSqlBackupKey1.key

```diff
@@ -0,0 +1,38 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
We should not check-in this file in repo. Please remove it from the PR. Also remove the second .key file that you have added in this PR.